src/components/NavBar/style.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+import { LinkColored } from '@/styles/text';
+
+export const NavBarContainer = styled.div`
+  /* top: 0;
+  left: 0;
+  position: absolute; */
+  width: 100%;
+  height: 78px;
+  background: ${COLORS.blueMid};
+  align-items: center;
+  z-index: 1000;
+  width: 100%;
+  box-shadow: 0px 4px 7px 0px rgba(0, 0, 0, 0.1);
+`;
+export const NavBarDiv = styled.div`
```

Comment:
`NavBarDiv` seems a little redundant as it shares a lot of properties with `NavBarContainer`. If you move over the `padding`, `display`, `justify-content`, and `flex-direction` to `NavBarContainer`, we can delete `NavBarDiv` while having the same effect on the navbar.